### PR TITLE
[VEN-2539]: add DAI market to core pool on ethereum and sepolia

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description
Multisig transaction to add DAI market to core pool on sepolia  : https://sepolia.etherscan.io/tx/0x1f524ad3ed27aa26903f907acc05ae2937e45bca932f1d34af086bdb9775a14e

Resolves VEN-2539

### CODE REVIEW

File: multisig/proposals/ethereum/vip-021/index.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+const CHAINLINK_DAI_FEED = "0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9";
+const CHAINLINK_STALE_PERIOD = 100 * 60;
+
+export const DAI = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
+export const vDAI = "0xd8AdD9B41D4E1cd64Edad8722AB0bA8D35536657";
+export const USDT = "0xdAC17F958D2ee523a2206206994597C13D831ec7";
```
Comment (chechu) [inline multisig/proposals/ethereum/vip-021/index.ts:14]: Not needed

File: multisig/simulations/ethereum/vip-021/index.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  });
+
+  describe("Post-VIP behavior", async () => {
+    before(async () => {
+      // This trasaction will be removed once Vtreasury on ethereum has enough funds
+      const impersonatedDaiHolder = await initMainnetUser(
+        "0x2D86e074C34e1FF2D33e8049Ee28e21Ce3A9Aa16",
+        parseUnits("1", 18),
+      );
+      await dai.connect(impersonatedDaiHolder).transfer(ethereum.VTREASURY, INITIAL_SUPPLY);
```
Comment (chechu) [inline multisig/simulations/ethereum/vip-021/index.ts:54]: DAI already available: https://etherscan.io/tx/0x7e7fff1d1a7398014c58f58be7aadb9d73a869a9f1db505d8ca2dd26068e3711

File: multisig/simulations/ethereum/vip-021/index.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      await checkVToken(vDAI, {
+        name: "Venus DAI (Core)",
+        symbol: "vDAI_Core",
+        decimals: 8,
+        underlying: DAI,
+        exchangeRate: parseUnits("1", 28),
+        comptroller: COMPTROLLER,
+      });
+    });
+  });
```
Comment (chechu) [inline multisig/simulations/ethereum/vip-021/index.ts:94]: Could we check the interest rate and the behaviour of the pool, using shared checks?

File: multisig/proposals/ethereum/vip-021/index.ts (hunk 1 above)
Comment (GitGuru7) [reply multisig/proposals/ethereum/vip-021/index.ts]: [33a7a2c](https://github.com/VenusProtocol/vips/pull/266/commits/33a7a2ca78d231ebaef71813c4c6985267b33c67)

File: multisig/simulations/ethereum/vip-021/index.ts (hunk 2 above)
Comment (GitGuru7) [reply multisig/simulations/ethereum/vip-021/index.ts]: [33a7a2c](https://github.com/VenusProtocol/vips/pull/266/commits/33a7a2ca78d231ebaef71813c4c6985267b33c67)

File: multisig/simulations/ethereum/vip-021/index.ts (hunk 3 above)
Comment (GitGuru7) [reply multisig/simulations/ethereum/vip-021/index.ts:101]: [33a7a2c](https://github.com/VenusProtocol/vips/pull/266/commits/33a7a2ca78d231ebaef71813c4c6985267b33c67)

File: multisig/proposals/sepolia/vip-019/index.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { sepolia } = NETWORK_ADDRESSES;
+
+export const CHAINLINK_DAI_FEED = "0x14866185B1962B63C3Ea9E03Bc1da838bab34C19";
+const CHAINLINK_STALE_PERIOD = 26 * 60 * 60;
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply multisig/proposals/sepolia/vip-019/index.ts]: https://github.com/VenusProtocol/vips/pull/266/commits/32421c325eab1784e8eb12220a371733ddef4ff1

File: multisig/proposals/ethereum/vip-021/index.ts [hunk 5]
```diff
@@ -0,0 +1,69 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+const CHAINLINK_DAI_FEED = "0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9";
+const CHAINLINK_STALE_PERIOD = 100 * 60;
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply multisig/proposals/ethereum/vip-021/index.ts]: https://github.com/VenusProtocol/vips/pull/266/commits/32421c325eab1784e8eb12220a371733ddef4ff1

**Review by chechu — APPROVED** [review]

File: multisig/proposals/ethereum/vip-021/index.ts
Comment (chechu) [inline multisig/proposals/ethereum/vip-021/index.ts:1]: I would move this multisig TX (and the sepolia one) to `022`, because it finally was at nonce 22

Comment (GitGuru7) [reply multisig/proposals/ethereum/vip-021/index.ts:1]: [e607acc](https://github.com/VenusProtocol/vips/pull/266/commits/e607acc06e966633867e1cefbc8770559d3ead70)

**Comment by chechu** [comment]
https://etherscan.io/tx/0xf2d385e717b4e9889f2d5b2821a4ede92e56b0d3f11fba6ca50ec1fffd5d2c7e

**Pull request merged**